libs/features/pages/home/src/lib/features-pages-home.module.ts

```diff
@@ -4,6 +4,7 @@ import { CommonModule } from '@angular/common';
 import { HomePageComponent } from './main/home/home-page.component';
 
 import { HomeRoutingModule } from './home.routing';
+import { ElementsLayoutCarouselModule } from '@elewa-website/elements/layout/carousel';
 
```

Comment:
No need to push this change. Any component that needs this will import it.

Reply:
corrected